api/models/UserPassword.coffee

```diff
@@ -0,0 +1,63 @@
+# UserPassword.coffee
+#
+# @description :: Accepts and validates incoming POST request for user password.
+# @docs        :: http://sailsjs.org/documentation/concepts/models-and-orm/models
+
+module.exports =
+  # Allow only whitelisted attributes.
+  schema: true
+
+  # Turn off migrations.
+  migrate: 'safe'
+
+  # Turn off auto fields.
+  autoCreatedAt: off
+  autoUpdatedAt: off
+  autoPK: off
+
+  # Validation rules.
+  attributes:
+    user_id:
+      type: 'string'
+      required: yes
+      hexadecimal: yes
+      defaultsTo: ""
+    email:
+      type: 'string'
+      required: yes
+      email: yes
+      defaultsTo: ""
+    mobile:
+      type: 'string'
+      required: yes
+      defaultsTo: ""
+    application_id:
+      type: 'string'
+    email_template:
+      type: 'string'
+
+  # Validate presence at least one of fields.
+  beforeValidate: (values, cb) ->
+    # The list of fields.
+    fields = ['user_id', 'email', 'mobile']
+
+    # Check if at least one field is present.
+    found = false
+    for field in fields
+      if values[field]? and !_.isEmpty values[field]
+        found = true
+        break
+
+    # Turn off presence validations when at least one fields is present,
```

Comment:
No, unfortunately, no. This is clearly a hack, but there's no functionality to do it in the right way. 

In addition, originally I was setting `this._validator.validations[field].required` to `false` instead of deleting it (which is also hackish, but more explicit). However, there's a bug in sails validation I described in the comment below: 

> It should be possible to just set required to false, but for some reason it turns off the rest of field rules.